src/components/VTabs/meta/VTabs.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs'
+
+import VTabs from '~/components/VTabs/VTabs.vue'
+import VTabPanel from '~/components/VTabs/VTabPanel.vue'
+import VTab from '~/components/VTabs/VTab.vue'
+
+<Meta title="Components/VTabs" component={VTabs} />
+
+export const Template = (args) => ({
+  template: `<VTabs v-bind="args"><template #tabs>
+<VTab id='1'>Tab1</VTab><VTab id='2'>Tab2</VTab><VTab id='3'>Tab3</VTab>
+</template>
+<VTabPanel id='1'>Page 1 content</VTabPanel>
+<VTabPanel id='2'>Page 2 content</VTabPanel>
+<VTabPanel id='3'>Page 3 content</VTabPanel>
+</VTabs>`,
+  component: { VTabs },
+  subcomponents: { VTab, VTabPanel },
+  setup() {
+    return { args }
+  },
+})
+
+# Tabs
+
+<Description of={VTabs} />
+
+<ArgsTable of={VTabs} />
+
+## Default tabs
+
+By default, tabs are bordered and use automatic activation of the tab panel when the tab is focused on,
+such as the attribution panels in Openverse.
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      label: 'Default tabs story',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Navigation
+
+`←` and `→`: changes the focused (and selected, if not manual) tab to previous or next. Tab navigation wraps, so if you click `→` on the last tab,
+you select the first tab.
+
+`tab`: navigate to the tablist and then to the selected tabpanel. The tabs use 'rowing index' to only allow to tab to the currently selected tab using keyboard.
+
+`home`, `end`: set the first or the last tab as focused (or selected).
+
+## Manual plain tabs
+
+To activate the tabs using keyboard, you need to focus on the tab you want by using the `Tab` key, and then click `Space` or `Enter`.
+
+These tabs also use the second available variant, `plain`.
```

Comment:
I wonder about this "manual" mode. From an accessibility standpoint, is it better or worse? Also, is there anything to indicate to the user that they need to make an additional action to activate the tab, versus the typical arrow key navigation?

Reply:
The W3 guidelines recommend using "manual" mode if you make a network request to display the tabpanel content: https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_selection_follows_focus

When I look at the network tab, I see that both requests for the feedback forms are made almost simultaneously, and each tab is  ~15kb. I wonder if it's worth the inconsistency of tab selection experience?

Reply:
Ah, okay! Thank you for sharing the document :slightly_smiling_face: It had not occurred to me that opening a tab could incur latency!

It's probably fine if it's part of the spec, I'd say! I was surprised that there wasn't an additional aria attribute to indicate the change in behavior to the user.

Reply:
Yes, that difference in behavior did make me think it's kind of inconsistent. But I didn't do anything about it :)